src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -178,7 +176,7 @@ public static String parseItemType(String itemType) throws ParseException {
         requireNonNull(itemType);
         String trimmedItemType = itemType.trim();
         if (!Item.isValidItemType(trimmedItemType)) {
-            throw new ParseException("Not a valid item type!");
```

Comment:
This deleted message appears three times:
-Line 315 in `AddCommandParserTest.java`
-Line 79 in `ViewCommandParserTest.java`, and
-Line 61 in` ListCommandParserTest.java`.

The message for parsing an invalid type will therefore no longer match the expected message.

Reply:
Fixed in the latest commit (9bb18d2)!

Reply:
Well done pal.